### PR TITLE
Added Surveys-crud

### PR DESCRIPTION
**Pull request by PetNamedEr**
Other changes:
-app.py requires  parameter for template_folder to change default ../templates-folder
-added index.html for tests

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,15 +1,11 @@
 from flask import Flask
 
 # Flask
-app = Flask(__name__)
+
+app = Flask(__name__, template_folder="c:/pts_feedback/views")
```
Comment (Arvinje) [inline app.py]: Remember some of us run this app on a Mac/linux machine. Use relative paths.

Comment (Arvinje) [reply app.py]: `views` would be enough.

Comment (PetNamedEr) [reply app.py]: Yeah, kind of knew there is some better way to do this. I tried set-variable first but it didn't work - but good that this works, I'll edit this to next version

**Comment by PetNamedEr** [comment]
1) with ID, is there some readily made function we are gonna use for incrementations?
2) How can I make validations, wtforms :)?
3) With invalid data (id = 23y for example) I get error and can go back to prev page without restarting the server. But I'll add error-handling

**Comment by Arvinje** [comment]
See the docs (SQLAlchemy's and flask's) for auto increment primary key creation (for ID) and the form validations. 
Regarding no.3, when I get the error, I cannot the form page again. Not sure if you're experiencing the same or not.

**Pull request merged**